src/cmd/group.go

```diff
@@ -278,13 +279,8 @@ var deleteGroupCmd = &cobra.Command{
 	ArgAliases: []string{"ID", "ALIAS"},
 	Run: func(cmd *cobra.Command, args []string) {
 		key := args[0]
-		if common.IsID(key) {
-			err := getClientGQL().DeleteGroup(key)
-			cobra.CheckErr(err)
-		} else {
-			err := getClientGQL().DeleteGroupWithAlias(key)
-			cobra.CheckErr(err)
-		}
+		err := getClientGQL().DeleteGroup(key)
```

Comment:
Does this stay functionality the same?

Reply:
Yes, [DeleteGroupWithAlias](https://pkg.go.dev/github.com/opslevel/opslevel-go/v2023@v2023.5.18#Client.DeleteGroup) is basically a wrapper that uses `DeleteGroup` under the hood